### ISSUE TITLE
List of models provided in link for model checkpoints

### ISSUE THREAD

**Issue by zainsarwar865** [facebookresearch/unlikelihood_training#5]
Hello,

Can you tell me which language models are available in the 16 GB file on this link  [https://dl.fbaipublicfiles.com/unlikelihood/checkpoints_v0.tar.gz](https://dl.fbaipublicfiles.com/unlikelihood/checkpoints_v0.tar.gz)
Is it just the gpt2-small or did you guys also evaluate on other sizes?

Thank you!

**Comment by stephenroller** [comment]
@wellecks @uralik

**Comment by uralik** [comment]
hey @zainsarwar865, the archive you mentioned consists of the fairseq LM checkpoints (find more details about models we used in section 6). These checkpoints are related with tables 1,2,3. 
We did not publish the gpt-2 checkpoints, but the training script one can use to finetune their own model together with the wikitext-103 dataset.

**Comment by zainsarwar865** [comment]
Thanks for the swift reply. So, if I understand correctly, the checkpoints in the archive are for an LM that you trained from scratch which is based on the Transformer architecture and has 16 layers each with 8 attention heads, an embedding dimension of 1024, and fully-connected dimension 4096? 
And as an aside, you fine-tuned the GPT2-small with your proposed loss function to show that the results were similar to what you achieved when you trained a model from scratch.

**Comment by uralik** [comment]
> So, if I understand correctly, the checkpoints in the archive are for an LM that you trained from scratch which is based on the Transformer architecture and has 16 layers each with 8 attention heads, an embedding dimension of 1024, and fully-connected dimension 4096?

yep!

> you fine-tuned the GPT2-small with your proposed loss function to show that the results were similar to what you achieved when you trained a model from scratch.

Yes, and more details about what metrics were more similar than other you can find in Appendix C (+ table 7)

**Comment by zainsarwar865** [comment]
Got it. Thanks a lot!
